distribution/docs/src/main/resources/content/_installing/high-availability-initial-setup-exceptions.adoc

```diff
@@ -0,0 +1,63 @@
+:title: High Availability Initial Setup Exceptions
+:type: subInstalling
+:status: published
+:parent: High Availability Initial Setup
+:project: ${branding}
+:order: 00
+
+==== {title}
+
+These steps are handled differently for the initial setup of a Highly Available Cluster.
+
+===== HTTP Proxy Integration
+
+In order to integrate with an HTTP proxy, the ${branding} node's system properties must be changed to publish the correct port to external systems and users.
+This must be done before installing the first ${branding} node.
+There are two internal port properties that must be changed to whatever ports the ${branding} will run on on its system.
+Then there are two external port properties that must be changed to whatever ports the HTTP proxy is forwarding traffic through.
+
+[WARNING]
+====
+Make sure that the HTTP proxy is already running and forwarding traffic on the chosen ports before starting the ${branding}.
+There may be unexpected behavior otherwise.
+====
+
+In the below example, the HTTP proxy is forwarding https traffic via 8993 and http traffic via 8181.
+The ${branding} node will run on 1111 for https and 2222 for http (on the virtual machine that it's hosted on).
+[source]
+----
+org.codice.ddf.system.httpsPort=8993
+org.codice.ddf.system.httpPort=8181
+org.codice.ddf.system.port=${org.codice.ddf.system.httpsPort}
+org.codice.ddf.system.internalHttpsPort=1111
+org.codice.ddf.system.internalHttpPort=2222
+----
+
+===== Identical Directory Structures
+
+The two ${branding} nodes need to be in identical directories on their corresponding systems.
+
+===== Highly Available Security Auditing
+
+A third party tool will have to be used to persist the logs in a highly available manner.
+
+* Edit the ``<INSTALL_HOME>/etc/org.ops4j.pax.logging.cfg` file to enable log4j2, following the steps in <<_enabling_fallback_audit_logging, Enabling Fallback Audit Logging>>.
+* Then put the appropriate log4j2 appender in `<INSTALL_HOME>/etc/log4j2.config.xml` to send logs to the chosen third party tool.
+See https://logging.apache.org/log4j/2.x/manual/appenders.html[Log4j Appenders].
+
+===== Highly Available NFS Server
+
+Whenever a shared directory is used, the NFS server being used should be highly available.
+
+===== Shared Storage Provider
+
+The storage provider must be in a location that is shared between the two ${branding} nodes and must be highly available.
+If hardening the Highly Available Cluster, this shared storage provider must be trusted/secured.
+One way to accomplish this is to use the default <<org.codice.ddf.catalog.content.impl.FileSystemStorageProvider,Content File System Storage Provider>> and configure it to point to a shared directory.
```

Comment:
🖊 "When using the default Content File System Storage Provider, configure and use a highly available shared directory" or something along those lines?